src/store/RelayDeferredFragmentTracker.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright 2013-2015, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule RelayDeferredFragmentTracker
```

Comment:
heads up that @steveluscher is working on a diff that will completely remove the need for the deferred tracker